js/apps/admin-ui/package.json

```diff
@@ -57,7 +57,7 @@
       ]
     },
     "build-lib": {
-      "command": "LIB=true vite build",
+      "command": "cross-env LIB=true vite build",
```

Comment:
The account console also sets environment variables in the same way, that should probably also be modified.

Reply:
Good idea.  I'll add that.